pkg/node/netconfig.go

```diff
@@ -76,9 +77,15 @@ func (n *NetworkConfiguration) Run(ctx context.Context, ready chan<- struct{}, s
 }
 
 func (n *NetworkConfiguration) addServiceIPLoopback() error {
-	link, err := netlink.LinkByName(loopbackInterface)
-	if err != nil {
-		return err
+	var link netlink.Link
+	var err error
+
+	link, err = netlink.LinkByName(ovnExternalBridge)
+	if _, ok := err.(netlink.LinkNotFoundError); ok {
```

Comment:
Who will create `br-ex`in the no-uplink mode in a new microshift deployment?
I wonder if this service address would always be assigned to loopback device.

Reply:
configure-ovs.sh will create `br-ex` with `ovs-vsctl` command instead of NM.

Reply:
In a fresh deployment, the service address will always be assigned to loopback device since this node network configuration runs before the InfrastructureServices controller (where CNI is started). Is it expected?

Reply:
The service address will be assigned to `lo` only if `br-ex` doesn't exist. It means the cluster is started without microshift-ovs-init.service. This is for the case that we use a non-OVN-K CNI.

Reply:
The details in this comment thread are an important part of the design. Are they written down somewhere? If not, could you add them in a comment in this function, please? You can use a follow-up PR, since this one looks like it is almost ready to be merged.